localisation/languages.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "languageName": "English",
+    "description": "NZ English (default)",
+    "code": "default",
```

Comment:
Is this something defined in here? Or this is not in use yet? I thought that the default language would be something defined per project.

Reply:
"default" just means -- use the "master" strings stored in the front-end, which will always be English.

But I've just realised I haven't made "disabling" the default actually work (disabling others does), so I should fix that (in front end)

Reply:
But no matter what, the "default" will always be the fallback when strings are missing. (unless we implement a cascading fallback system like I commented in the front-end PR, but that's actually not that trivial to do in an efficient way)

Reply:
Okay, I've given this a bit more thought, and I think this is how is should work. See if you agree:

- the "default" is the front-end strings, and these are ONLY used when keys are missing from Language files
- so even an "English" language file will still be fetched from server, even if it has identical content to the "default" set in front-end (though we can over-ride specific terms if we want to)
- that way "English" won't be "special" in any way (other than being the fallback), and will just be another editable language file on the server.
- So there won't be a "default" language code, "default" will just be "fallback".

Reply:
Done 92d277f3  (and 0471f59e in front-end)

Makes much more sense now, thanks for pointing it out.

Reply:
It'll start with whatever language is set as the default in preferences. It's currently in `config.ts` in the front-end, but I've moved it to a common "preferences.json" in the [later PR](#595 )

But you can still switch language from the login screen before you even start.